README.md

```diff
@@ -89,12 +89,12 @@ The following methods can be used with this module. All methods can be called by
 ### generateMnemonic()
```

Comment:
I think (or at least from what I can tell? let me know if I'm wrong) you've changed the `generateMnemonic()` method only take a wordcount argument, but forgot to fix it in the readme.